net/gce.sh

```diff
@@ -305,13 +298,27 @@ else
   sshPrivateKey="$netConfigDir/id_$prefix"
 fi
 
+defaultZone="$(cloud_DefaultZone)"
+
 case $cloudProvider in
 gce)
+  customMemoryGB="$(cloud_DefaultCustomMemoryGB)"
+  if [[ "$tmpfsAccounts" = "true" ]]; then
+    defaultZone="us-west2-a" # There's some resource limit on us-west1 ATM
```

Comment:
Incidentally, I tried running this and `nvidia-tesla-p100` is not available in that zone.

Reply:
I dropped the workaround since it seems like the issue is both config dependent and transient.